cmd/serve.go

```diff
@@ -65,16 +70,21 @@ func serve() {
 		logger.Fatal("Error initializing service broker: %s", err)
 	}
 
-	username := os.Getenv("SECURITY_USER_NAME")
-	password := os.Getenv("SECURITY_USER_PASSWORD")
+	username := viper.GetString("api.user")
+	password := viper.GetString("api.password")
 
 	credentials := brokerapi.BrokerCredentials{
 		Username: username,
 		Password: password,
 	}
 
 	// init api
+	logger.Info("Serving", lager.Data{
+		"port":     viper.GetString("api.port"),
+		"username": username,
+	})
+
 	brokerAPI := brokerapi.New(serviceBroker, logger, credentials)
 	http.Handle("/", brokerAPI)
-	http.ListenAndServe(":"+os.Getenv("PORT"), nil)
+	http.ListenAndServe(":"+viper.GetString("api.port"), nil)
```

Comment:
Instead of retrieving "api.port" twice, consider doing it once and assigning to a variable, similar to username and password.